conf/recipe.yaml

```diff
@@ -55,3 +55,24 @@ Manifests:
           exit 100
   - Platform:
       os: windows
+    Artifacts:
+      - URI: s3://use-local-provided-artifact/custom.zip
+        Unarchive: zip
+        Permission:
+          execute: ALL
+          read: ALL
```

Comment:
remove this, we don't have it for other platforms

Reply:
Oops, removed